### PR TITLE
added nitesh to contributors

### PR DESCRIPTION
**Pull request by sudonitesh**

### CODE REVIEW

**Comment by palash25** [comment]
@itsmepiyush2 check for grammatical errors

and try to review all the other upcoming contributor PRs

File: _contributors-app/sudonitesh.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ - Will update soon. 
+
+project:
+- |
+    - [Text Editor](https://github.com/sudonitesh/text-editor): A hosted text-editor that can be used to share the typed text through url and get html code from typed text
+- |
+   - [Wiki Viewer](https://github.com/sudonitesh/fcc-wikipedia): Wikipedia Viewer.
+
+volunteer_experience: |
+ - Third in team based GenHax hackathon. Made a web-app to digitalise garbage-collection of the country.
```
Comment (itsmepiyush2) [inline _contributors-app/sudonitesh.md]: A better statement: "I participated in a team based hackathon "GenHax" where we secured the third position. We made a web-app to help digitalise India's garbage collection."

**Comment by knrt10** [comment]
@sudonitesh please remember to make one commit only. Didn't you read [meta](https://github.com/p-society/meta). Please squash your commits

**Comment by sudonitesh** [comment]
@knrt10  I've squashed the commits locally, Now what to do? 

nitesh@Nitesh-PC MINGW64 ~/Desktop/p-society.github.io (profile)
$ git rebase -i profile
Successfully rebased and updated refs/heads/profile.
nitesh@Nitesh-PC MINGW64 ~/Desktop/p-society.github.io (profile)
$ git log --oneline
bb0b431 (HEAD -> profile) add nitesh to contributors after update
6037255 (origin/master, origin/HEAD) added social media icons to contact page
27c2edd added regex test
....
nitesh@Nitesh-PC MINGW64 ~/Desktop/p-society.github.io (profile)
$ git log --author=sudonitesh
commit bb0b431756ce2f093b22444dacf994fd03436a08 (HEAD -> profile)
Author: sudonitesh <nktosni@gmail.com>
Date:   Mon Aug 27 22:58:39 2018 +0530

    add nitesh to contributors after update

    added nitesh to contributors

    updated volunteer_experience

**Comment by sudonitesh** [comment]
I think its done. Please check

File: _contributors-app/sudonitesh.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+githubUsername: sudonitesh
+speciality: |
+
+  - Developing Responsive Full Stack Web Apps
+
+  - Open Source
+
+github_url: https://github.com/sudonitesh
+linkedIn_url: https://www.linkedin.com/in/sudonitesh/
+twitter_url: https://twitter.com
```
Comment (sibasish14) [inline _contributors-app/sudonitesh.md]: @sudonitesh give your correct Twitter url.

Comment (sudonitesh) [reply _contributors-app/sudonitesh.md]: Have you seen me on twitter?!! I don't have one..

**Comment by sibasish14** [comment]
Then remove the link. Why do you want to redirect someone to Twitter when you don't have one?😉

**Comment by palash25** [comment]
@sudonitesh good work you know how to squash now.

But why do you repeat the same shortlog twice in your commit message. Please read how to make good commit message the link is in the Meta README

**Comment by sudonitesh** [comment]
@palash25 ya sure i'll . do i need to make any change in this pr?

**Comment by palash25** [comment]
Just told you remove the second occurence of the shortlog and push, then I will merge

Also shibasis gave you a review and you haven't addressed it.

**Comment by palash25** [comment]
Which second commit? You only have one

**Comment by palash25** [comment]
Change this message 

```
added nitesh to contributors
added nitesh to contributors

updated volunteer_experience
```

to this

```
added nitesh to contributors

updated volunteer_experience
```

And also remove the twitter link if you don't have a twitter

**Comment by sudonitesh** [comment]
@palash25 please review

**Pull request merged**